### ISSUE TITLE
Add logic in call assignment query to prefer calling people outside work hours

### ISSUE THREAD

**Issue by saikat** [Bernie-2016/ground-control#367]

**Comment by saikat** [comment]
The solution here would be:

1) Get a list of valid timezones where it is between 6 and 9 pm.
2) If no such valid timezones exist, use all valid timezones to query
3) If valid timezones in non-work hours exist, try to find a person in those timezones.  If no people, then do the query again using all valid timezones.

**Comment by nkammah** [comment]
@saikat I'm just getting familiar with the code - does the above issue pertains to https://github.com/Bernie-2016/ground-control/blob/master/src/backend/data/schema.js#L431-L436

where instead of looking for valid times between 9am to 9pm, we'd restrict to 6pm to 9pm ? (and then implement 2) and 3) ? 

Also as a side note (to fix), in dev mode, the validOffset array does not quite look right
```
and "zip_codes"."timezone_offset" in ('-10', '-9', '-8', '-7', '-6', '-5', '-4', '-10', '-9', '-8', '-7', '-6', '-5', '-4')
```

**Comment by saikat** [comment]
@nkammah Yes, exactly!   And for dev mode -- we could clean up the repetition.  In practice it doesn't make a difference.
